cas/incident-report.md

```diff
@@ -1,169 +1,387 @@
+### Summary
+<!---
 
-To create the full audit incident report, copy the markdown template below and fill out each section according to the following instructions and requirements:
+**Expectations:**
 
-1. The Finding section should contain a short description of the nature of each finding. This provides context for the reader to understand the nature of the non-compliance identified by the Auditor.
-2. The Root Cause Analysis section must contain a detailed analysis of the conditions which combined to give rise to the issue. It is unusual for an incident to have a single root cause; often there must be a confluence of several issues such as a software bug, insufficient checks, and a malformed request. Make sure that all contributing causes are identified and described, including noting when they first arose and how they avoided detection until they were discovered or identified.
-3. The Action Items section must contain a list of remediation items that were or will be undertaken to ensure that similar incidents do not reoccur in the future. Note that it is not sufficient for these action items to simply stop this incident, they must create additional protections to prevent future incidents. Each Action Item should state:
-   - A short description of the action that was or will be taken.
-   - A classification of whether the action will help _Prevent_ future incidents, _Mitigate_ the impact of future incidents, or _Detect_ future incidents. CA Owners are encouraged to propose action items in all three categories, with an emphasis on prevention and mitigation.
-   - A date by which the action item was or will be completed.
+- This section SHOULD clearly address the following:
+     - **Incident description**: [A short description of the nature of the issue. This provides just enough context for new readers to understand the details of the incident.]
+     - **Relevant policies**: [Describe the policies and corresponding sections that result in this problem being diagnosed as an incident.]
+ 
+- If the report was created by the corresponding CA Owner, this section MUST also answer the following question:
+     - **When is the Full Incident Report expected by?**: [DATE within 2 weeks of the incident’s initial disclosure (i.e., either to the CA Owner, or Bugzilla)]
 
-If the audit includes multiple findings, the audit incident report should include a separate Finding, Root Cause Analysis, and Action Items section for each finding. 
+-->
+```
 
-### Audit Incident Report Template
+#### Full Incident Report
 
 ```markdown
-## Audit Incident Report
+## Full Incident Report
 
-### Finding #1
+### Summary
+<!---
```

Comment:
Is the content within the HTML comment block supposed to be reviewed, or should it be ignored for now?

Reply:
We believe this is now overcome by events, given field descriptions and expectations have been removed from the reporting templates and into a new section in #187.